lib/seek/config_setting_attributes.yml

```diff
@@ -231,7 +231,7 @@ omniauth_github_client_id:
 omniauth_github_secret:
   encrypt: true
 ga4gh_trs_api_enabled:
-custom_metadata_enabled: false
+custom_metadata_enabled:
```

Comment:
we don't want this as a configuration, unless the views are updated to honour it, otherwise things will break
The previous line is an old artifact where it was added and then removed, choosing instead to be determined by whether any custom metadata is configured, and in the future be able to turn of individual custom metadata definitions

Reply:
The reason for making this change is that I removed the line  `return if supported_type == "CustomMetadata"`
from [app/models/custom_metadata_type.rb](https://github.com/seek4science/seek/pull/1564/files/9ef544c92b9334156287c353cd0ef9cd1f475c96#diff-5597508804483c32920f9c0f88bad920d703e4bf6c35f8a9d65db7da9e547b95)

I introduced the new supported type, "CustomMetadata," to distinguish between custom metadata directly related to ISA items and linked custom metadata. Removing this line resulted in the failure of CustomMetadataType creation when a linkedCustomMetadata type was present.